### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by JuanOfMany**
Juan set up some messy code. 

Xinyuan cleaned up that messy code.

Db connection and MVC separation code done.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,28 +4,47 @@
...
+    "css-loader": "^6.7.1",
+    "dotenv": "^16.0.1",
+    "express": "^4.18.1",
+    "node-pg": "^1.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
+    "style-loader": "^3.3.1",
+    "styled-components": "^5.3.5",
     "zustand": "^4.0.0"
```
Comment (addisonhernandez) [inline package.json:32]: :bear:

File: server/controllers/messageController.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const db = require('../../database/postgres.js');
+
+/**
+ * @route /api/message
+ * @method GET
+ * @desc get a message
```
Comment (addisonhernandez) [inline server/controllers/messageController.js:6]: JSDOC?! :fire: :fire: :fire:

File: src/index.js [hunk 3]
```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
```
Comment (addisonhernandez) [inline src/index.js:2]: If you're using React Router...

Take a look at `React.Lazy`, so you don't have to import all these components on the initial page load.

**Review by addisonhernandez — COMMENTED** [review]
Nice skeleton! :skull_and_crossbones:

**Review by maria6417 — APPROVED** [review]
LGTM! 
So organized and looks easy to start working on!

**Pull request merged**